v21.2/global-tables.md

```diff
@@ -84,6 +84,14 @@ Because the `test` database does not specify a [survival goal](multiregion-overv
 
 For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).
 
+## Troubleshooting
+
+### High follower read latency on global tables
+
+Reads on multiregion global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 
```

Comment:
Just the comment that github added - change this from multiregion to multi-region

Reply:
thanks Andrew - pushed a fix!